tests/integration-tests/tests/common/mpi_common.py

```diff
@@ -28,6 +28,8 @@ def _test_mpi(
     scaledown_idletime=None,
     verify_scaling=False,
     partition=None,
+    verify_pmix=None,
```

Comment:
do we need to have this as a separate if block variable?  Could we instead call this function once and run all the tests?